python-package/xgboost/callback.py

```diff
@@ -325,9 +325,11 @@ class EarlyStopping(TrainingCallback):
     maximize :
         Whether to maximize evaluation metric.  None means auto (discouraged).
     save_best :
-        Whether training should return the best model or the last model. This is only
-        supported with tree methods. Also, the `cv` function doesn't return a model, the
-        parameter is not applicable.
+        Whether training should return the best model or the last model. If set to
+        `True`, it will only keep the boosting rounds up to the detected best iteration,
+        discarding the ones that come after. This is only supported with tree
+        methods. Also, the `cv` function doesn't return a model, the parameter is not
```

Comment:
This one could also mention the 'gblinear' part.

Reply:
Done.